clients/benchmarks/CMakeLists.txt

```diff
@@ -231,9 +231,48 @@ if (NOT WIN32)
 endif()
 
 if (NOT WIN32)
-  if (BUILD_WITH_ROCTX)
-    target_link_libraries(rocsparse-bench PRIVATE -lroctx64)
+  set(ROCTX_PATH "" CACHE STRING "Path to the roctx library (directory containing libroctx64.so)")
+
+  find_path(ROCTRACER_INCLUDE_DIR
```

Comment:
roctracer is eventually going to be replaced by the rocprofiler-sdk. There is also roctx provided by the rocprofiler-sdk. At some point, you'll need to support that roctx as well.

You can find that version of roctx via `find_package(rocprofiler-sdk-roctx PATHS ${ROCM_PATH} /opt/rocm)` and linking to the `rocprofiler-sdk-roctx::rocprofiler-sdk-roctx` target.

[It's been used in hipfort for a while.](https://github.com/ROCm/hipfort/blob/rocm-6.4.1/lib/CMakeLists.txt#L127-L132)